star-wars-app/Pages/People/PeopleViewModel.swift

```diff
@@ -13,8 +13,8 @@ import Moya
 class PeopleViewModel {
     private let repository = PeopleRepository()
     
-    var peopleObservable: Observable<People> {
-        return repository.peopleObservable.asObservable()
+    var peopleObservable: Observable<[Person]> {
```

Comment:
If you wanted just do this quickly, cause you know that you won't develop this project much that's fine, but... It'd be better to create cell view model, that has only the fields, that your view needs. This may seem useless when you're at the initial stage of the project and you're model is designed exactly to have all the data that you need in your view - no more, no less. However your view may evolve, sometimes you'll need some data from your app's services, and then your first move would be probably to create a struct that contains a Person and some other data... Generally your model and view can change from totally different reasons and it's better for view to be separated from your models from the beginning. I don't even apply YAGNI in this case.